tests/shared/cli/run_smoke_test.py

```
@@ -135,14 +135,14 @@ def start_servers(server_dir, config_path):
         try:
             with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
                 s.settimeout(1.0)
-                if s.connect_ex(("127.0.0.1", 6450)) == 0:
+                if s.connect_ex(("127.0.0.1", 6200)) == 0:
                     break
         except Exception:
             pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)